Solaris 11.4 since SRU69 implements preadv and pwritev.

Accidentally in libuv the preadv / pwritev emulation was brought back, since
Node.js required to support older systems. This happened in commit:

https://github.com/libuv/libuv/commit/cc23e204d76688053c63fd1698435bc4b68aab6a

The emulation is slightly different. It no longer hardcodes which system
supports preadv and pwritev, instead it implements autodetection using dlsym().
Once the commit lands in official libuv release, it will use dlsym() to
discover real preadv / pwritev implementation in Solaris 11.4 SRU69+ and will
use it instead of emulating the functionality.

This libuv patch can be removed in a future libuv upgrade (perhaps 1.49).

diff -rup libuv-v1.48.0.orig/src/unix/fs.c libuv-v1.48.0.new/src/unix/fs.c
--- libuv-v1.48.0.orig/src/unix/fs.c	2024-02-07 21:20:07.000000000 +0100
+++ libuv-v1.48.0.new/src/unix/fs.c	2024-08-28 11:14:34.721864095 +0200
@@ -84,7 +84,6 @@
 
 #if defined(__CYGWIN__) ||                                                    \
     (defined(__HAIKU__) && B_HAIKU_VERSION < B_HAIKU_VERSION_1_PRE_BETA_5) || \
-    (defined(__sun) && !defined(__illumos__)) ||                              \
     (defined(__APPLE__) && !TARGET_OS_IPHONE &&                               \
      MAC_OS_X_VERSION_MIN_REQUIRED < 110000)
 #define preadv(fd, bufs, nbufs, off)                                          \
